css/index.css

```diff
@@ -126,6 +155,103 @@ p {
     padding-right: 0;
 }
 
```

Comment:
Good job changing only what was needed in each individual box and letting the parent take care of the rest of the styling.